packages/dashboard-frontend/src/services/bootstrap/index.ts

```diff
@@ -463,4 +463,19 @@ export default class Bootstrap {
     }
     return -1;
   }
+
+  private setFavicon() {
+    const base64data =
+      this.store.getState().clusterConfig.clusterConfig.dashboardFavicon?.base64data;
```

Comment:
@tolusha Could you add a selector into [selectors.ts](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/store/ClusterConfig/selectors.ts)?

Something just like this:
```
export const selectDashboardFavicon = createSelector(
  selectState,
  state => state.clusterConfig.dashboardFavicon,
);

```